Plugins/CoreAPI/server/graphql/schema/schema.graphql

```diff
@@ -0,0 +1,137 @@
+type CoreAPIConfiguration {
+  hookURL: String
+}
+
+input ProjectInput {
```

Comment:
At some point we ought to come through here and document/decide if all these fields are needed, but it's fine for now

Reply:
agreed